tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/BulkheadFutureTest.java

```diff
@@ -77,126 +83,121 @@ public void beforeTest(final ITestContext testContext) {
      * Tests that the Future that is returned from an asynchronous bulkhead
      * method can be queried for Done OK before and after a goodpath .get()
      */
-    @Test()
+    @Test
     public void testBulkheadMethodAsynchFutureDoneAfterGet() {
-
         Checker fc = new FutureChecker(SHORT_TIME);
         Future<String> result = null;
 
         try {
             result = bhBeanMethodAsynchronousDefault.test(fc);
         }
         catch (InterruptedException e1) {
-            Assert.fail("Unexpected interruption", e1);
+            fail("Unexpected interruption", e1);
         }
 
-        Assert.assertFalse(result.isDone(), "Future reporting Done when not");
+        assertFalse(result.isDone(), "Future reporting Done when not");
         try {
             String r;
-            Assert.assertTrue("GET.".equals(r = result.get()), r);
-            Assert.assertTrue("GET.GET_TO.".equals(r = result.get(1, TimeUnit.SECONDS)), r);
+            assertEquals(r = result.get(), "GET.", r);
+            assertEquals(r = result.get(1, TimeUnit.SECONDS), "GET.GET_TO.", r);
```

Comment:
I don't like this `assertEquals(r = result.get(), "GET", r)` structure.

Can we change it to `assertEquals(result.get, "GET");` and get rid of `r`?

I'd hope that `assertEquals` gives a sensible error message like `Expected "GET" but was "something else"` so passing the result as the message shouldn't be necessary.

Reply:
done